lib/src/components/Sections/index.vue

```diff
@@ -1451,7 +1534,7 @@ export default {
         }
 
         const URL =
-          `${this.$sections.serverUrl}/project/${this.$sections.projectId}/page/${this.pageName}`;
+          `${this.$sections.serverUrl}/project/${this.$sections.projectId}/page/${encodeURIComponent(this.pageName)}`;
```

Comment:
I think it's the same here (when you want to load a page to render it) and you should use the pagePath not the pageName

Reply:
Same as above